libsolidity/experimental/analysis/TypeInference.cpp

```diff
@@ -300,6 +300,7 @@ bool TypeInference::visit(InlineAssembly const& _inlineAssembly)
 		*_inlineAssembly.annotation().analysisInfo,
 		m_errorReporter,
 		_inlineAssembly.dialect(),
+		{},
```

Comment:
I think it's better to use the empty string literal where possible. For reading it makes it easier to recognize which parameter it might be. And for writing will cause an error if you accidentally use it for a non-string argument.
```suggestion
		"",
```

It also stands out more as a corner case - actually, only while writing this comment I started wondering whether it even makes sense to use an empty name :). `{}` does not draw my attention because it looks to me like an empty container which is normal. On the other hand, hard-coding a string, even an empty one, is something that requires more attention.